.github/config/chunks.yaml

```diff
@@ -43,19 +43,19 @@ chunks:
       - compaction/compaction-job-creation-lambda
       - compaction/compaction-tracker
       - compaction/compaction-core
+      - compaction/compaction-datafusion
```

Comment:
This can't go in the compaction chunk because Rust isn't enabled in this chunk. We really need the Rust build to only be in one chunk because it takes too long.